docs/topics/http/views.txt

```diff
@@ -212,13 +212,13 @@ Asynchronous views
 
 As well as being synchronous functions, views can also be asynchronous
 functions (``async def``). Django will automatically detect these and run them
-in an asynchronous context. You will need to be using an asynchronous (ASGI)
-server to get the full power of them, however.
+in an asynchronous context. However, you will need to use an asynchronous
```

Comment:
I see we are replacing `asynchronous` with `async` at many places in the changes. Is there any specific reason we are not doing that here? 

Also if we do change it could the heading be `Async views` instead of `Asynchronous views` as it's there in [async doc](https://github.com/django/django/blob/ffde4d5da88da6ed40ffc090503896d329bc67f8/docs/topics/async.txt#L25).

Reply:
I initially didn't convert this because it's only in the async topic guide we write "asynchronous (async)" to make it clear it's a contraction. But I've re-edited this section to do so too.

@andrewgodwin I replaced "async functions" with "async coroutines" because that's all the Python documentation seems to refer to the result of `async def` as.

Reply:
I don't like the Python documentation for async stuff, so I'd rather we were more explicit (nobody knows what a coroutine is). How about "asynchronous coroutines/functions"?

Reply:
I've reset it to "functions." We use the word "coroutine" in the async topic guide, I think that's enough.